### PR TITLE
Calc logic

### PR DESCRIPTION
**Pull request by DilsherB**
In this PR:
1. Get data from API
2. Display data from API on page
3. Loading state showing
4. Error catched too
5. useEffect hook used
6. API key implemented using .env file

### CODE REVIEW

File: src/components/APIkey.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+const apiKey = "eqaKx38o0Fg1f/rZqOA/sg==epV9zWMwWAegkqnF";
+export default apiKey;
```
Comment (codepantha) [inline src/components/APIkey.js]: - [x] Storing API keys or other sensitive information on a git repository should be avoided at all costs. Please make use of environment variables in a `.env` file.
Please use this [resource](https://dev.to/felixdusengimana/environment-variables-in-reactjs-139l) if you need some help 😎

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @DilsherB 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Used GitFlow
✔️ Fetch data from API 📌 
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/APIkey.js (hunk 1 above)
Comment (DilsherB) [reply src/components/APIkey.js]: @codepantha thank you for your review.
but I am unable to do it. Please guide me. I have gone through the link  you provided but unable to import it.

Comment (codepantha) [reply src/components/APIkey.js]: @DilsherB I'm sorry for the difficulties you are experiencing. Please reach me on slack or we could have a call on zoom 📹

**Review by CollinsTatang — COMMENTED** [review]
Hi @DilsherB, 

While you made a great effort in this project. Unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

The project you have submitted does not have the requested changes by the previous reviewer.

- [ ] Please only submit again for review when you have been able to complete the requested changes
- [ ] Please your commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). Kindly correct that in your next commit massage.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/APIkey.js (hunk 1 above)
Comment (DilsherB) [reply src/components/APIkey.js]: Hi @codepantha 
I couldn't find you on slack. please reach me so that we can go through this.

Comment (codepantha) [reply src/components/APIkey.js]: Okay. I just sent you a message

Comment (DilsherB) [reply src/components/APIkey.js]: Hi @codepantha 
I called you on zoom and waited there for 43 minutes but you didn't appear. I can understand you may be busy but please come up and let it finish.

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED 💯 

Hi @DilsherB , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Great use of .env to store API key
:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file and PR

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**